PurchasesTests/Mocks/MockDeviceCache.swift

```diff
@@ -74,7 +74,6 @@ class MockDeviceCache: DeviceCache {
     // MARK: offerings
 
     var cacheOfferingsCount = 0
-    var cachedOfferingsCount = 0
```

Comment:
This `var` was duplicated. I chose `cacheOfferingsCount` considering the naming rule used in the rest of the mocks.

Reply:
I _think_ the context on that one is that we used to have this mock duplicated locally in a few places, then I unified them while trying to unify the components and I must have missed this one, thanks for the cleanup!!